app/views/books/_form.html.erb

```diff
@@ -0,0 +1,29 @@
+<%= form_with model: @book do |form| %>
+  <div>
+    <%= form.label :title, "Title" %>
+    <%= form.text_field :title, required: true, placeholder: "Enter book title" %>
```

Comment:
Placeholder text is usually used as an example of what the input should contain, rather than additional or supplemental instructions.

Since some screen readers don't announce placeholders and since in these cases the placeholders replicate the labels, I'd suggest removing them.

Some sources say placeholders are harmful due to their low contrast or ability to make it seem like a field is already filled (especially if you increase the contrast), W3C has some [example research](https://www.w3.org/WAI/GL/low-vision-a11y-tf/wiki/Placeholder_Research)